README.md

```diff
@@ -100,6 +105,37 @@ General security best practices apply:
   * Ultimately, the security of a system depends on the capabilities of its administrator.
   * Care should be taken when installing new software; only install from official sources that the developers indicate on their official website/github/etc.
 
+# Threat Modeling
+
+The first and most important step for security and privacy is to create a [threat model](https://www.owasp.org/index.php/Application_Threat_Modeling). You need to understand your adversaries in order to defend against them. Each person will have their own needs so everyone's threat model will be different. Threat models tend to evolve over time as our situation changes, so be sure to periodically reassess your threat model.
+
+## Step 1: Identify assets
+
+This is probably a lot of things: your phone, your laptop, passwords stored on your devices, internet browsing history, etc. Make a list starting with the most important assets to protect. You can put them in categories based on how important they are: public, sensitive, or secret.
+
+## Step 2: Identify adversaries
+
+Define whom you are defending against. Start by defining the motivation they might have to attack your assets. [Financial gain](https://www.verizon.com/business/resources/reports/dbir/) is a big motivator for many attackers, for example.
+
+## Step 3: Identify capabilities
+
+In order to counter your adversaries, you'll need to understand what they're capable of and what they're not capable of. Rank adversaries from totally usophisticated to very advanced. For example, a common thief is not very sophisticated; they will likely be stopped by basic things like simply having a password and drive encryption on your device. A very advanced adversary like a state actor might require fully turning off your device when not in use to clear the keys from RAM and a long diceware password.
+
+## Step 4: Identify mitigations
+
+Now is when you decide the best way to counter each threat. You might avoid writing passwords down on paper so your roommate can't find them or you might encrypt the drive on your computer so a thief can't get data from it. It's important to balance security and usability; every mitigation should counter some capability of your adversaries, otherwise you might be making your life inconvenient for little to no gain. If you can't think of any more capabilities your adversaries might have and you've implemented mitigations for them all, your work is done.
+
+Here's an example of the type of table you should make for each asset you want to protect:
+
+| Adversary | Motivation | Capabilities | Mitigation |
+| --- | --- | --- | --- |
+| Family | See private chats or browsing history | Close proximity; can see screen or watch type in password | Use biometrics, use privacy screen, keep phone locked when not using it |
```

Comment:
Family as an adversary seems unnecessarily hostile here, even if it is in the realm of possibilities. I wonder if we can use a more neutral tone here - maybe "Insider"? We could also add another column for examples of that adversary, up to you.

Reply:
Yeah maybe. Just meant it like, someone who is in close proximity to you that might have physical access like a roommate or something. Not really sure of a word for that lol.